tests/upgrade_path/restoreToFirstBoot.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euo pipefail
+
+set -x
+
+IMAGE_DIR_NAME=$1
+
+mount -t proc /proc
+mount -t tmpfs tmpfs /run
+mount --bind /host/${IMAGE_DIR_NAME}/docker /var/lib/docker
+
+dockerd -l warn &
+
+sleep 3
+
+docker system prune -f
+
+rm -rf /host/${IMAGE_DIR_NAME}/rw
+touch /host/${IMAGE_DIR_NAME}/platform/firsttime
```

Comment:
@saiarcot895 what are the things today taken care of ONLY during firstime?

Reply:
Installation of the platform-specific support package, and copying configs from `/host/old_config` to `/etc/sonic`. I'm not sure if there's others.